docroot/profiles/favrskov/modules/features/other/feeds_job_importer/feeds_job_importer.feeds_tamper_default.inc

```diff
@@ -11,7 +12,7 @@ function feeds_job_importer_feeds_tamper_default() {
   $export = array();
 
   $feeds_tamper = new stdClass();
-  $feeds_tamper->disabled = FALSE; /* Edit this to true to make a default feeds_tamper disabled initially */
+  $feeds_tamper->disabled = TRUE; /* Edit this to true to make a default feeds_tamper disabled initially */
```

Comment:
@mnakov 
Can you please confirm that this is correct and tamper plugin should be disabled? As I can see there is a regex which should trim styling tag in description field.

Reply:
Yes, I confirm. This tamper was not used at all. If you look at the current settings in https://favrskov.dk/admin/structure/feeds/job_importer/tamper you will see this tamper is not attached at all. 
![image](https://user-images.githubusercontent.com/9681642/55069793-84caba00-508d-11e9-81ba-eb8104b90af5.png)